frontend/components/containers/annotation/Seq2seqContainer.vue

```diff
@@ -36,7 +36,17 @@ export default {
     }
   },
 
+  created() {
+    this.updateSearchOptions({
+      offset: this.offset
```

Comment:
There is not an offset variable.

Reply:
@Hironsan, sorry, my mistake. Should I include "mapGetters('pagination', ['offset'])" in both?
I mean... do you think the change in PR is in the right way?

Reply:
I think the bug(https://github.com/doccano/doccano/issues/887) is very important so I reverted https://github.com/doccano/doccano/pull/868. Therefore, https://github.com/doccano/doccano/issues/887 should have been resolved.

The below is a comment for "Should I include "mapGetters('pagination', ['offset'])" in both?"
- Yes. But the code is duplicated so I think the layouts/annotation.vue will be a better place.

Reply:
I guess 868 is as important as this one. Maybe #868 should be applied and this bug should have a different approach, haven't it?